snippets/assert-attribute-must-be-set.js

```diff
@@ -0,0 +1,48 @@
+/* Usage of this snippet:
+ *   In the case when you want to confirm an element having a specified attribute.
+ *
+ *   Change the values written below.
+ *    selector: A string of a selector to pinpoint the element.
+ *    attrName: A string of the attribute which you want to assert its value.
+ *    errMessage:
```

Comment:
弊社のコード規約的には極力変数名は略さないようにしています
```suggestion
 *   errorMessage:
```